accumulo-handler/src/java/org/apache/hadoop/hive/accumulo/AccumuloStorageHandler.java

```diff
@@ -369,11 +369,7 @@ public void preCreateTable(Table table) throws MetaException {
           tableOpts.create(idxTable);
         }
       }
-    } catch (AccumuloSecurityException e) {
-      throw new MetaException(StringUtils.stringifyException(e));
-    } catch (TableExistsException e) {
-      throw new MetaException(StringUtils.stringifyException(e));
-    } catch (AccumuloException e) {
+    } catch (AccumuloSecurityException | TableExistsException | AccumuloException e) {
```

Comment:
In a previous PR (https://github.com/apache/hive/pull/3478) we opted to use the LOG + throw pattern as an alternative of removing `stringifyException` acknowledging advantages and disadvantages of it. Why we don't want to do the same here?